packages/unmock-node/src/__tests__/runner.test.ts

```diff
@@ -1,16 +1,17 @@
 import axios from "axios";
 import * as path from "path";
 import { Service, transform, UnmockPackage } from "unmock-core";
-import { runner } from "..";
 import NodeBackend from "../backend";
 const { withCodes } = transform;
+import jestRunner from "../../../unmock-runner/src/jestRunner";
```

Comment:
Hmm, I don't think one should refer to other Lerna packages via "../../../" but with something like `import { runner} from "unmock-runner"`. Jest will figure out that `unmock-runner` should point to the local package.